config/test-urls.js

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+// some of the alexa US top 50 sites and subpages
+module.exports = [
+  { url: 'https://www.google.com/?gws_rd=ssl',
```

Comment:
I don't explicitly call this out anywhere, but one of my goals with this file is to make it really easy to generate specific training sets, like clusters of pages around a search, or big sets of social or media-focused pages. I opted for `.js` over `.json` because I hate how you can never cleanly insert comments in a JSON file; comments are nice.